dependencies/README.md

```diff
@@ -87,3 +89,74 @@ The tool allows users to choose the scope of entry point classes:
   - **All classes in the input class path**: when reachability check is off, then
     all static linkage errors from all classes in the classpath, regardless of the reachability,
     are reported.
+
+<a name="maven_dependency_graph"></a>
+## Maven Dependency Graph
+
+A Maven dependency graph is a graph data structure where
+- Node: a node is a Maven artifact identified as `groupId:artifactId[:classifier]:version`, where
+  `classifier` is optional.
+- Edge: an (directed) edge is a dependency between Maven artifacts. A dependency from a Maven
+  artifact (source of the dependency) to another artifact (target of the dependency) is defined
+  in `dependencies` tags in pom.xml of a Maven artifact.
+  
+  A dependency has a boolean attribute `optional` and a string attribute `scope`,
+  among other properties listed in [POM Reference: Dependencies][1].
+
+  Self-loops or parallel edges are not possible.
+
+### Graph Construction
+
+Given a list of Maven artifacts, a Maven dependency graph is defined in the following manner:
+
+- Create a graph with nodes of the Maven artifacts. They are called initial nodes.
```

Comment:
I don't quite follow this paragraph. In particular edges are added from what *to what*?